tests/bluetooth/tester/src/l2cap.c

```diff
@@ -66,6 +66,10 @@ static void connected_cb(struct bt_l2cap_chan *l2cap_chan)
 	if (!bt_conn_get_info(l2cap_chan->conn, &info)) {
 		switch (info.type) {
 		case BT_CONN_TYPE_LE:
+			ev.mtu_remote = chan->le.tx.mtu;
+			ev.mps_remote = chan->le.tx.mps;
+			ev.mtu_local = chan->le.rx.mtu;
+			ev.mps_local = chan->le.rx.mps;
```

Comment:
Don't you need byte order conversions here? Isn't `ev` protocol order whereas `chan->le` is host order?

Reply:
indeed, I missed that